tests/playwright-test/web-server.spec.ts

```diff
@@ -429,6 +429,37 @@ test(`should suport self signed certificate`, async ({ runInlineTest, httpsServe
   expect(result.exitCode).toBe(0);
 });
 
+test('should send Accept header', async ({ runInlineTest }, { workerIndex }) => {
+  const port = workerIndex + 10500;
+  let acceptHeader: string | undefined | null = null;
+  const server = http.createServer((req: http.IncomingMessage, res: http.ServerResponse) => {
+    if (acceptHeader === null) acceptHeader = req.headers.accept;
+    res.end('<html><body>hello</body></html>');
+  });
```

Comment:
There's a built-in `server` fixture for testing, so that should be used instead of creating a server inline here. See https://github.com/microsoft/playwright/blob/9795a4ba60479fc75de7187e681ba9e90891bb54/tests/page/page-goto.spec.ts#L72 for an example.

Reply:
Thanks for the feedback, I'll adjust it accordingly. However, I just copied it from other tests in the same file. Should the other tests also be adjusted to use the `server` fixture`?

Reply:
Oops—thanks for pointing that out.

> Should the other tests also be adjusted to use the server fixture?

I'll have to take a closer look before cleaning those up, but I've merged your change.